user_stories.md

```diff
@@ -0,0 +1,11 @@
+# Пользовательские истории.
+
+1. Как пользователь, я хочу иметь возможность создавать сокращенные ссылки без регистрации на сервисе (возможно, с ограниченным временем работы ссылок), чтобы их было удобнее использовать в текстах, так как они[ccылки] будут со значительно меньшим числом символов.
```

Comment:
Does user create a short link or service?

If you are talking about temporal constraints, you should specify the way it will work.
And word "maybe". There can't be "maybe" in a user story. "Do. Or do not. There is no try". =)

Reply:
> Does user create a short link or service?
I don't quite understand this question, from our description it's obvious: "я хочу иметь возможность [создавать (кого? что?) сокращенные ссылки] без регистрации на сервисе", service is not a subject of an action so user doesn't create any service according to this description.

Reply:
My fault. Does user or does service generate shot links?

Reply:
Good question. We decided to allow both options. Add it to story.

Reply:
Service generate short link. Optionally, user can set constraints for symbols in generated link or specify desired result.